README.md

```diff
@@ -364,6 +364,14 @@ def handler(event, context):
     context.iopipe.mark.measure('expensive operation')
 ```
 
+The trace plugin can also trace your HTTP/HTTPS requests automatically. To enable this feature, set `auto_http` to `True` or set the `IOPIPE_TRACE_AUTO_HTTP` environment variable. For example:
```

Comment:
Change env var to `IOPIPE_TRACE_AUTO_HTTP_ENABLED`

Reply:
@kolanos This is still an open comment